collector/vrrp.go

```diff
@@ -60,48 +48,57 @@ type VrrpInstanceStats struct {
 }
 
 type vrrpCollector struct {
-	logger log.Logger
+	logger       log.Logger
+	descriptions map[string]*prometheus.Desc
 }
 
 // NewVRRPCollector collects VRRP metrics, implemented as per the Collector interface.
 func NewVRRPCollector(logger log.Logger) (Collector, error) {
-	return &vrrpCollector{logger: logger}, nil
+	return &vrrpCollector{logger: logger, descriptions: getVRRPDesc()}, nil
+}
+
+func getVRRPDesc() map[string]*prometheus.Desc {
+	labels := []string{"proto", "vrid", "interface", "subinterface"}
+	stateLabels := append(labels, "state")
+
+	return map[string]*prometheus.Desc{
+		"vrrpState":       colPromDesc(vrrpSubsystem, "state", "Status of the VRRP state machine.", stateLabels),
+		"adverTx":         colPromDesc(vrrpSubsystem, "adverTx_total", "Advertisements sent total.", labels),
```

Comment:
adverTx sounds cryptic to me. "advertisements_send_total" and using "received" for Rx would be clearer. In case you change that, please put it into a separate commit.

Reply:
I agree. I will fix this in a subsequent PR.